cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/util/YamlUtils.java

```diff
@@ -0,0 +1,45 @@
+package cloud.foundry.cli.crosscutting.util;
+
+import cloud.foundry.cli.crosscutting.beans.ApplicationBean;
+import cloud.foundry.cli.crosscutting.beans.ApplicationManifestBean;
+import cloud.foundry.cli.crosscutting.beans.ServiceInstanceSummaryBean;
+import cloud.foundry.cli.crosscutting.beans.SpaceDevelopersBean;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.nodes.Tag;
+import org.yaml.snakeyaml.representer.Representer;
+
+import java.util.HashMap;
+
+public class YamlUtils {
```

Comment:
The name of this class isn't too nice. It's, after all, mainly a factory. I think we have some `*Creator` elsewhere, maybe use consistent naming.

Docstring missing.